style/css/react-toggle-style.css

```diff
@@ -0,0 +1,141 @@
+.react-toggle {
```

Comment:
Do you know what this css is for? I did a search for react-toggle styling in source and didn't find any components that are using this.

Reply:
I can remove later if it's not necessary.